### DISCUSSION TITLE
Does proxbox support more than one proxmox cluster?

### DISCUSSION THREAD

**Discussion by dragona-Ocient** [emersonfelipesp/netbox-proxbox#69] [Q&A]
Does proxbox support more than one proxmox cluster?  

I have it installed with a single cluster but looking ahead we'd like to have multiple clusters of proxmox that are in netbox.  Does proxbox support that setup?  

It looks like the configuration expects only one host/cluster at this time.

**Comment by RashitVoroshilov** [comment]
The same question. 
Just tried to solve it - can't find the way to write it in configuration.py

**Comment by emersonfelipesp** [comment]
There's already [one issue open about it](https://github.com/netdevopsbr/netbox-proxbox/issues/33) and the roadmap includes multi-cluster support! Just have to wait, but if you are willing to help with code, would be great!
